packages/ag-charts-types/src/api/initialStateOptions.ts

```diff
@@ -4,10 +4,21 @@ import type { AgStateSerializableDate } from './stateTypes';
 export interface AgInitialStateOptions {
     /** The initial set of annotations to display on the chart. */
     annotations?: AgAnnotation[];
+    /* The initial chart type. */
+    chartType?: AgInitialStateChartType;
     /** The initial zoom state. */
     zoom?: AgInitialStateZoomOptions;
 }
 
+export type AgInitialStateChartType =
+    | 'candlestick'
+    | 'hollow-candlestick'
+    | 'ohlc'
+    | 'line'
+    | 'step-line'
+    | 'hlc'
+    | 'high-low';
```

Comment:
Ideally this would just be `AgInitialStateChartType = Preset['chartType']`, but the docs would just show `Preset['chartType']`, so it needs to be recreated in full.

Reply:
I think I would opt for one of (in order of precedence):
- Fixing the doc generation.
- Duplication, but with a compile-time check that the types match (see `__VERIFY_THEME_OVERRIDES`).

Reply:
Fixed, it will now expand `type AgInitialStateChartType = AgPriceVolumeChartType` into:

<img width="588" alt="Screenshot 2024-11-01 at 11 56 05" src="https://github.com/user-attachments/assets/664b44eb-bdde-4a7a-b33c-a0d7a06ce430">

So if we add new presets to this state then it shouldn't be a breaking change.